distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalogValidation.java

```diff
@@ -0,0 +1,1124 @@
+/**
```

Comment:
For maintainability, can you please add some of this Github comment in the Javadoc? Seems like other developers could use the guidance on why to add to this class. Maybe `testMetacardValidation` is more appropriate?